test/Sentry.Tests/Sentry.Tests.csproj

```diff
@@ -43,4 +43,10 @@
     <PackageReference Include="Microsoft.Extensions.Configuration.Binder" Version="8.0.1" />
   </ItemGroup>
 
+  <ItemGroup Condition="'$(TargetFramework)' == 'net9.0' OR '$(TargetFramework)' == 'net8.0'">
```

Comment:
Can this be something that will automatically add future versions? something akin to NET8_OR_GREATER but for msbuild conditions? I think we have this somewhere in there repo

Reply:
https://github.com/getsentry/sentry-dotnet/blob/1bf37620fe5955defb7eca8aa1218e3a969d9413/src/Sentry/Platforms/Native/buildTransitive/Sentry.Native.targets#L23